src/twisted/python/deprecate.py

```diff
@@ -258,8 +258,14 @@ def _appendToDocstring(thingWithDoc, textToAppend):
     elif len(docstringLines) == 1:
         docstringLines.extend(["", textToAppend, ""])
     else:
-        spaces = docstringLines.pop()
+        trailer = docstringLines[-1]
+        spaces = ""
+        if not trailer.strip():
+            # On Python 3.13 the docstring is already stripped.
+            # For older Python version we keep the trailer.
```

Comment:
I'm not sure this sentence is quite accurate?

Reply:
I am also not sure what was going in there :( 
This was done 2 months ago, and now I no longer remember what was the issue here.

I will try to review and update the docstring